draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1547,6 +1547,112 @@ In order to verify countersignatures of messages in a group, a signature checker
+
+  * https://github.com/eclipse-californium/californium
+
+  * http://eclipse.dev/californium/
+
+* Implementation's level of maturity: prototype
+
+* The implementation supports:
+
+  * The group mode and the pairwise mode.
+  * Mapping of public keys for the curve Ed25519 into Montgomery coordinates to use with X25519.
+  * The following COSE encryption algorithms: AES-CCM-16-64-128, AES-CCM-16-128-128, AES-CCM-16-64-256, AES-CCM-16-128-256, AES_CCM-64-64-128, AES-CCM-64-128-128, AES-CCM-64-64-256, AES-CCM-64-128-256, A128GCM, A192GCM, A256GCM, ChaCha20/Poly1305, A128CBC, A192CBC, A256CBC.
+  * The following HKDF algorithms: HKDF SHA-256 (identified as the COSE Algorithm "HMAC 256/256") and HKDF SHA-512 (identified as the COSE Algorithm "HMAC 512/512").
+  * The following COSE signature algorithms: ECDSA with curves P-256, P-384, and P-521, as well as EdDSA with curve Ed25519.
+  * The following COSE key agreement algorithms: ECDH-SS + HKDF-256 and ECDH-SS + HKDF-512, both of which using either keys of COSE Key Type "EC2" with the curve P-256, P-384, and P-521, or keys of COSE Key Type "OKP" key with X25519.
+  * The following authentication credential format: CWT Claims Sets (CCSs).
+
+* Version compatibility: From version -23 onwards.
+
+* Licensing: according to the same dual license of Eclipse Californium, i.e., according to the "Eclipse Distribution License 1.0" and the "Eclipse Public License 2.0". See:
+
+  * https://github.com/eclipse-californium/californium/blob/main/LICENSE
+  * https://www.eclipse.org/org/documents/edl-v10.php
+  * https://www.eclipse.org/legal/epl-2.0/
+
+* Contact information: Rikard Höglund - rikard.hoglund@ri.se
+
+* Information last updated on: 2025-02-06
+
+## Implementation \#2 # {#sec-implementation-2}
+
+* Implementation's name: aiocoap
+
+* Available at: https://codeberg.org/aiocoap/aiocoap
+
+* Description: A Python CoAP library with support for multiple transports and security mechanisms. The library provides also utility programs.
+
+* Implementation's level of maturity: Stable support for OSCORE; Group OSCORE is WIP.
+
+* The implementation supports:
+
+  * The group mode and the pairwise mode.
+  * Mapping of public keys for the curve Ed25519 into Montgomery coordinates to use with X25519.
+  * The following COSE encryption algorithms: 1-3, 10-13, 24, 30-33, -65531
+  * The following HKDF algorithms: HKDF SHA-256, -384, -512.
+  * The following COSE signature algorithms: EdDSA on Ed25519, ECDSA w/ SHA-256 on P-256
+  * The following COSE key agreement algorithms: ECDH on P-256 and curve25519.
+  * The following authentication credential format: currently n/a (user provides pairs of credentials and keys)
+
+* Version compatibility: -23
+
+* Licensing: MIT
+
+* Implementation experience: Mostly smooth; the differentiation between the regular and the group AEAD algorithm was tedious and error prone (because Group OSCORE largely relies on OSCORE extension points that were anticipated, but that was not).
```

Comment:
```suggestion
* Implementation experience: Mostly smooth; the differentiation between the regular and the group AEAD algorithm, and more generally finding the right parameters to input into the (abstracted) KDF part, was tedious and error prone (because Group OSCORE largely relies on OSCORE extension points that were anticipated, but that was not).
```